ui/src/components/Drawer/components/Keyboard/index.vue

```diff
@@ -67,30 +67,6 @@ const keyboardList = reactive<KeyboardItem[]>([
       writeDataToTerminal('Stop');
     }
   },
-  // {
-  //   icon: Save,
-  //   label: t('Save'),
-  //   keywords: ['Command/Ctrl', 'S'],
-  //   click: () => {
-  //     writeDataToTerminal('Save');
-  //   }
-  // },
-  // {
-  //   icon: ClipboardPaste,
-  //   label: t('Paste'),
-  //   keywords: ['Command/Ctrl', 'V'],
-  //   click: () => {
-  //     writeDataToTerminal('Paste');
-  //   }
-  // },
-  // {
-  //   icon: Undo2,
-  //   label: t('Undo'),
-  //   keywords: ['Command/Ctrl', 'Z'],
-  //   click: () => {
-  //     writeDataToTerminal('Undo');
-  //   }
-  // },
   {
     icon: ArrowUp,
     label: t('UpArrow'),
```

Comment:
I'm sorry, but there seems to be an error in this piece of JavaScript code.

This is your example:

```typescript
{
    icon: Save,
    label: t('Save'),
    keywords: ['Command/Ctrl', 'S'],
    click: () => {
       writeDataToTerminal('Save');
        }
      },
```

Please change it to:

```json
{
    "icon": "Icon",
    "label": t("Title"),
    "keywords": ["Key word"],
    "click": function() { writeDataToTerminal('Function') }
}
```